tests/FSharp.Data.GraphQL.IntegrationTests.Server/WebSocketMiddleware.fs

```diff
@@ -0,0 +1,166 @@
+namespace FSharp.Data.GraphQL.IntegrationTests.Server
```

Comment:
Same here. It does not matter that namespace will be StarWars if we link that

Reply:
Will keep iterating on this in 2023/02/LHK-607-Introspection_Query .